src/metabase/types.clj

```diff
@@ -22,8 +22,14 @@
 (derive :entity/ProductTable :entity/GenericTable)
 (derive :entity/SubscriptionTable :entity/GenericTable)
 (derive :entity/EventTable :entity/GenericTable)
+
 (derive :entity/GoogleAnalyticsTable :entity/GenericTable)
 
+(derive :entity/GHCommitsTable :entity/GenericTable)
```

Comment:
GH is too cryptic here ... as we add data types, there will be multiple domains where "GH" could come from.

Reply:
Agree, this is temporary to make it work with what we have. #9564 outlines a more elegant approach, where not every entity type needs to be global.